math/const.go

```diff
@@ -14,12 +14,15 @@ func newDecimal(s string) *decimal.Big {
 	return x
 }
 
-const constPrec = 100
+const (
+	constPrec             = 100
+	defaultExtraPrecision = 3
+)
 
 var (
 	_E    = newDecimal("2.718281828459045235360287471352662497757247093699959574966967627724076630353547594571382178525166427")
 	_Ln10 = newDecimal("2.302585092994045684017991454684364207601101488628772976033327900967572609677352480235997205089598298")
-	_Pi   = newDecimal("3.141592653589793238462643383279502884197169399375105820974944592307816406286208998628034825342117067")
+	_Pi   = newDecimal("3.141592653589793238462643383279502884197169399375105820974944592307816406286208998628034825342117")
```

Comment:
Why'd you truncate _Pi?

Reply:
Short answer: based on GDA round to even the last digit was rounded incorrectly
Long answer:
I just looked at that bit again and realized I could have done it differently.  I thought the Pi function "somehow" used _Pi and extended it, however, that's not the case at all.  Instead of truncating to a zero so that any extension should be fine, I could have/should have just corrected the last digit (from 7 to 8).  The Pi function only uses _Pi if it is <= constPrec(100).  Sorry about that, I really should have caught that, some how when I looked at the code originally I got in my head _Pi was somehow seeding your CF, but looking now it's clear I was wrong.

For ref below is pi from http://www-groups.dcs.st-and.ac.uk/history/HistTopics/1000_places.html
3.14159265358979323846264338327950288419716939937510
  582097494459230781640628620899862803482534211706**7**9
  82148086513282306647093844609550582231725359408128

Proposed changed:
That said, I think the changes I made to lines 22 and 61 should be undone and instead change only the last digit in _Pi to 8.

Reply:
This is my first time using github for a code review before a merge.  How do you want to handle changes that need to be made due to the code review? 
I assume you'd want me to make changes then put in for a new PR?
If I make any changes and push to this branch will it mess up this review process? (Other tools I've used pushing changes while in code review would cause a problem.)

Reply:
No worries. There's a fair bit of code that isn't super clear but exists solely for performance reasons.

WRT how we can do the code review, I think what I'll do is go through it myself (locally) then push my changes back up and you can ask questions, tell me mistakes I've made, etc. I think that way will work best because you've a much deeper understanding of this than I do!